src/main/java/onehundredandfour/RestApis.java

```diff
@@ -0,0 +1,141 @@
+package onehundredandfour;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Rest Api class
+ *
+ */
+@RestController
+@RequestMapping("/v1") 
+@Api(value="restapis", description="All controllers available for the 104 application.")
+public class RestApis {
+	private static final String version = "0.0.1";
+	List<Card> deck = GameHelper.createDeck();
+	List<Player> players = new ArrayList<>();
+	Game game = null;
+	
+	private static final String template = "Hello, the version of this application is: %s!";
+
+	/**
+	 * About service, showing the version of the app.
+	 *
+	 */
+	@ApiOperation(value="Details about the 104 application.")
+
+@ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Succesfully retrieved message.")
+}
+)
+    @RequestMapping(value = "/about", method = GET)
+    public static String getVersion() {
+        return new String(String.format(template, version));
+    }
+    
+    /**
+     * CreatePlayer, will create a player each call.
+     * @throws Exception when exceeding the player limit
+     *
+     */
+    @RequestMapping(value = "/createPlayer", method = GET)
+    public Player createPlayer() throws Exception {
+    GameHelper.shuffleDeck(deck);
+    if(deck.size() >= 10) {
```

Comment:
this way a player is able to join, even when the game has started already

Reply:
created immutable list, however it is still possible to call start game endpoint again, so not solved yet

Reply:
in one of the future changes we should implement a way to choose the number (default 10 now) of cards in hand.